dcat/index.html

```diff
@@ -1872,6 +1876,94 @@ <h4>Property: was generated by</h4>
         </section>
 
     </section> <!-- end class Dataset -->
+<section id="Class:DatasetSeries">
+    
+        <h3>Class: DatasetSeries</h3>
```

Comment:
Keeping coherence with other sections, the name of the class in the header is written in English.
```suggestion
        <h3>Class: Dataset Series</h3>
```